planner/core/planbuilder.go

```diff
@@ -538,7 +539,7 @@ func (b *PlanBuilder) buildAdmin(as *ast.AdminStmt) (Plan, error) {
 	}
 
 	// Admin command can only be executed by administrator.
-	b.visitInfo = appendVisitInfo(b.visitInfo, mysql.SuperPriv, "", "", "")
+	b.visitInfo = appendVisitInfo(b.visitInfo, mysql.SuperPriv, "", "", "", ErrPrivilegeCheckFail)
```

Comment:
How about to pass a `nil` error to `appendVisitInfo()` in the first step, then fix the error message by passing the right error to `appendVisitInfo()` one by one in the following steps?

Does all the function call of `appendVisitInfo()` should pass an error?

Reply:
Sure, I can pass `nil` instead.  Yes, all instances of `appendVisitInfo()` have been modified to pass an error.